src/Plot.js

```diff
@@ -570,12 +576,12 @@ export default class Plot extends Viz {
 
     const y2Transform = `translate(-${this._margin.right}, ${this._margin.top + topOffset})`;
     const y2Group = elem("g.d3plus-plot-y2-axis", {parent, transition, enter: {transform: y2Transform}, update: {transform: y2Transform}});
-
+    const timeMargin = xTime || x2Time ? xOffset - this._xTest.padding() : 0;
```

Comment:
this logic needs to be part of the `xDifference` variable when it's defined, as it directly effects `this._padding`, which in turn effects the positioning of any legend beneath the axis (so that it is centered on the axis itself). @cnavarreteliz please investigate why this needs to be different for time axis, because it is still not clear to me why this works (even though it does work).

Reply:
The only difference between Plot with/without `time()` is `.ticks(xTicks)`, where its value is `undefined` when `time()` is defined.

Reply:
great code condensing here @cnavarreteliz!